vercel/resource_edge_config_item.go

```diff
@@ -0,0 +1,260 @@
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/hashicorp/terraform-plugin-log/tflog"
+	"github.com/vercel/terraform-provider-vercel/client"
+)
+
+// Ensure the implementation satisfies the expected interfaces.
+var (
+	_ resource.Resource              = &edgeConfigItemResource{}
+	_ resource.ResourceWithConfigure = &edgeConfigItemResource{}
+)
+
+type edgeConfigItemResource struct {
+	client *client.Client
+}
+
+func (r *edgeConfigItemResource) Metadata(ctx context.Context, req resource.MetadataRequest, resp *resource.MetadataResponse) {
+	resp.TypeName = req.ProviderTypeName + "_edge_config_item"
+}
+
+func (r *edgeConfigItemResource) Configure(ctx context.Context, req resource.ConfigureRequest, resp *resource.ConfigureResponse) {
+	// Prevent panic if the provider has not been configured.
+	if req.ProviderData == nil {
+		return
+	}
+
+	client, ok := req.ProviderData.(*client.Client)
+	if !ok {
+		resp.Diagnostics.AddError(
+			"Unexpected Resource Configure Type",
+			fmt.Sprintf("Expected *client.Client, got: %T. Please report this issue to the provider developers.", req.ProviderData),
+		)
+		return
+	}
+
+	r.client = client
+}
+
+// Schema returns the schema information for an edgeConfigToken resource.
+func (r *edgeConfigItemResource) Schema(_ context.Context, req resource.SchemaRequest, resp *resource.SchemaResponse) {
+	resp.Schema = schema.Schema{
+		Description: `
+Provides an Edge Config Item.
+
+An Edge Config is a global data store that enables experimentation with feature flags, A/B testing, critical redirects, and more.
+
+An Edge Config item is a value within an Edge Config.
+`,
+		Attributes: map[string]schema.Attribute{
+			"edge_config_id": schema.StringAttribute{
+				Description:   "The name of the key you want to add to or update within your Edge Config.",
+				Required:      true,
+				PlanModifiers: []planmodifier.String{stringplanmodifier.RequiresReplace()},
+			},
+			"team_id": schema.StringAttribute{
+				Optional:      true,
+				Computed:      true,
+				Description:   "The ID of the team the Edge Config should exist under. Required when configuring a team resource if a default team has not been set in the provider.",
+				PlanModifiers: []planmodifier.String{stringplanmodifier.RequiresReplaceIfConfigured(), stringplanmodifier.UseStateForUnknown()},
+			},
+			"key": schema.StringAttribute{
+				Description:   "The ID of the Edge Config store.",
+				Required:      true,
+				PlanModifiers: []planmodifier.String{stringplanmodifier.RequiresReplace()},
+			},
+			"value": schema.StringAttribute{
+				Description: "The value you want to assign to the key.",
+				Required:    true,
+			},
+		},
+	}
+}
+
+type EdgeConfigItem struct {
+	Key          types.String `tfsdk:"key"`
+	Value        types.String `tfsdk:"value"`
+	TeamID       types.String `tfsdk:"team_id"`
+	EdgeConfigID types.String `tfsdk:"edge_config_id"`
+	Token        types.String `tfsdk:"token"`
```

Comment:
Token isn't defined in the schema above.

Reply:
I misinterpreted the documentation originally here, and assumed that we'd need to create a token to use the resource but it seems that's not the case.